benchmarks/ols/datasets/simulated.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+
+from benchopt.base import BaseDataset
+
+
+class Dataset(BaseDataset):
+
+    name = "Simulated"
+
+    # List of parameters to generate the datasets. The benchmark will consider
+    # the cross product for each key in the dictionary.
+    parameters = {
+        'n_samples, n_features': [
+            (500, 1000),
+            (200, 5000)]
```

Comment:
for OLS I think it's not really relevant to test with n_features > n_sample

Reply:
agreed. I switch the values between the two.